packages/main/src/DateRangePicker.ts

```diff
@@ -381,13 +380,6 @@ class DateRangePicker extends DatePicker implements IFormInputElement {
 
 		return "";
 	}
-
-	/**
-	 * @override
-	 */
-	get _calendarPickersMode() {
```

Comment:
The function is an override from datepicker that sets lets a range to be set only in the day picker, since we are imlementing the other views, this is no longer necessary